integration_tests/generics_array_02.py

```diff
@@ -16,21 +16,21 @@ def add_float(x: f32, y: f32) -> f32:
 
 def g(n: i32, a: T[n], b: T[n], **kwargs):
   r: T[n]
-  r = empty(n)
+  r = empty(n, dtype=object)
```

Comment:
This is fine for now to get it to compile. Good catch. This we will have to fix later, since this should really be just:
```
r: T[n] = empty(n, dtype=T)
```
But that requires cooperation of numpy and generics. We are improving the generics in LFortran, once we are done and have a reasonably good design, we'll update LPython as well.